test/performance/comm/low-level/many-to-one.ml-keys

```diff
@@ -1,2 +1,2 @@
-Execution time =
-Performance (mOps/sec) =
+Execution time per source =
```

Comment:
Note that this key renaming will requiring editing the .dat files, which is pretty annoying/cumbersome. It's possible if the renaming is important, but if it's not I would try to avoid it. 

See internal chapel-docs/bestPractices/updatingDatFiles.rst and [util/devel/updateDatFiles.py](https://github.com/chapel-lang/chapel/blob/master/util/devel/updateDatFiles.py)